### DISCUSSION TITLE
Allow typing to (briefly) extend the timer

### DISCUSSION THREAD

**Discussion by Lucretiel** [proginosko/LeechBlockNG#352] [Ideas]
I've had it happen where I'm in the middle of typing a reply to something, and the block kicks in, erasing everything I wrote. It'd be nice if the timer can't go below, say, 20 seconds (enough time to briefly proofread then click “post”) if you're actively typing in a page.

**Comment by proginosko** [comment]
Good suggestion. Let me think about the best way to implement this.
